app/components/activity-graph/component.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+// Register `activityGraph` component, along with its associated controller and
+// template
+angular.
+  module('activityGraph').
+  component('activityGraph', {
+    templateUrl: 'activity-graph/template.html',
+    controller: ['$scope', '$http', 'activitiesSearch', 
+    function ActivityGraphController($scope, $http, activitiesSearch) {
+      $scope.options = {
+            chart: {
+                type: 'stackedAreaChart',
+                height: 450,
+                margin : {
+                    top: 20,
+                    right: 20,
+                    bottom: 30,
+                    left: 40
+                },
+                x: function(d){return d[0];},
+                y: function(d){return d[1];},
+                useVoronoi: false,
+                clipEdge: true,
+                duration: 100,
+                useInteractiveGuideline: true,
+                xAxis: {
+                    showMaxMin: false,
+                    tickFormat: function(d) {
+                        return d3.time.format('%x')(new Date(d));
+                    }
+                },
+                yAxis: {
+                    tickFormat: function(d){
+                        return d3.format(',.2f')(d);
+                    }
+                },
+                zoom: {
+                    enabled: true,
+                    scaleExtent: [1, 10],
+                    useFixedDomain: false,
+                    useNiceScale: false,
+                    horizontalOff: false,
+                    verticalOff: true,
+                    unzoomEventType: 'dblclick.zoom'
+                }
+            }
+        };
+
+        $scope.data = [
```

Comment:
What is this? Please move them to constant, if this data is not being updated by someone!